spark-common/src/main/js/src/spark-common-templates.js

```diff
@@ -123,10 +123,42 @@ const appFullscreenContaineriFrame = function({id, src, createOptions, className
     `;
 };
 
+/**
+ *
+ * @param targetId
+ * @param text
+ * @returns {string}
+ */
+
+const inlineDialogTrigger = function({targetId, text}){
+    return `
+        <a data-aui-trigger aria-controls="${targetId}" style="cursor: pointer; color: inherit;">${text}</a>
+    `;
+};
+
+/**
+ *
+ * @param id
+ * @param createOptions
+ * @returns {string}
+ */
+const appBootstrapContainerInlineDialogWithiFrame = function({id, src, createOptions, className}) {
```

Comment:
This camel case form of the word iframe looks really really weird, but since it has been used in the same form at other places in this file I guess it's fine / consistently weird. 😉